<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CloudFlare</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icon.ico</normaloff>img/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgb(47, 47, 47);
color:rgb(200, 200, 200)
}
QTabWidget::pane {
	border: 1px solid rgb(128, 128, 128);
	border-radius: 3px;
}
QTabBar::tab 
{
    background: #48555E;
    color: white;   
	padding:5px;
    border: 1px solid #000;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
}
QTabBar::tab:selected, 
QTabBar::tab:hover 
{
    border-top-color: #1D2A32;
    border-color: #40494E;
    color: black;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #C1D8E8, stop: 1 #F0F5F8); 
}
QLineEdit, QComboBox, QTextBrowser{
	background-color: rgb(85,85,85);
	border-color: rgb(0, 0, 0);
	border-width : 1px;
	border-style:inset;
	padding: 5px;
	border-radius:3px;
}
QPushButton{
	background:rgb(102, 102, 102);
	color:rgb(255, 255, 255);
	border-radius:3px;
	padding-top:5px;
	padding-bottom:5px;
	padding-right:10px;
	padding-left:10px;
	border-color: rgb(0, 0, 0);
	border-width : 1px;
	border-style:inset;
}
QPushButton:hover{
	background-color:rgb(128, 128, 128);
}
QPushButton:pressed{
	background-color:#A4A4A4;
	color:#000
}
QLabel{
	border:0px;
}
QCheckBox{
	border:0px;
}
QTextBrowser{
	color: rgb(230, 230, 230);
	padding:10px;
	background-image: url(template/img/logo.png);
	background-repeat: no-repeat;
  	background-attachment: fixed;
 	background-position: center;
}
QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 5px;
}
QGroupBox::indicator {
    width: 13px;
    height: 13px;
}
QGroupBox#groupBoxNotifications::indicator:unchecked, QGroupBox#groupBoxNotifications::indicator:checked {
	image: url(template/img/notification.png);
}
QGroupBox#groupBoxSettings::indicator:unchecked, QGroupBox#groupBoxSettings::indicator:checked {
	image: url(template/img/settings.png);
}

QGroupBox#groupBoxBrowsers::indicator:unchecked, QGroupBox#groupBoxBrowsers::indicator:checked {
	image: url(template/img/browser.png);
}

QGroupBox#groupBoxTelegram::indicator:unchecked, QGroupBox#groupBoxTelegram::indicator:checked{
	image: url(template/img/telegram.png);
}

QGroupBox#groupBoxAccountSettings::indicator:unchecked, QGroupBox#groupBoxAccountSettings::indicator:checked{
	image: url(template/img/user.png);
}

QMenuBar {
    background-color: rgb(40,40,40);
}
QMenuBar::item {
    padding: 4px 8px;
    background: transparent;
    border-radius: 4px;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: rgb(60,60,60);
	color:rgb(200,200,200);
}

QMenuBar::item:pressed {
    background: #888888;
	color: #000;
}
QHeaderView::section, QHeaderView{
	background: #48555E;
	border: 0px;
	padding-left:3px;
}
QHeaderView::item{
	border: 0px;
}
QTableCornerButton::section {
    background: #48555E;
}
QTableWidget{
	border: 1px solid gray;
	border-radius:5px;
}
QTableWidget#tableWidgetPlanetList{
	border: 2px solid gray;
	border-radius:5px;
}
QTextBrowser#textBrowserWhoisNameservers, QTextBrowser#textBrowserWhoisStatus{
	margin:0px;
	padding:0px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <attribute name="icon">
      <iconset>
       <normaloff>img/dns.png</normaloff>img/dns.png</iconset>
     </attribute>
     <attribute name="title">
      <string>DNS</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetDomainList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>181</width>
        <height>461</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScrollMargin">
       <number>17</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::DashLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Domain Name</string>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/http.png</normaloff>img/http.png</iconset>
       </property>
      </column>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tableWidgetDnsRecords">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>691</width>
        <height>511</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Content</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Proxy Status</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="lineEditSearchDomain">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>182</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Search or Add Domain</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClearCache">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>86</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clear Cache</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddDomain">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>135</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>7</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Add New Domain</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>343</x>
        <y>10</y>
        <width>5</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRecordType">
      <property name="geometry">
       <rect>
        <x>711</x>
        <y>10</y>
        <width>69</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <item>
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AAAA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CNAME</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TXT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SRV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CAA</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButtonAddRecord">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Record</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>699</x>
        <y>10</y>
        <width>5</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSearchRecord">
      <property name="geometry">
       <rect>
        <x>356</x>
        <y>10</y>
        <width>334</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="placeholderText">
       <string>Search Record</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUnderAttack">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>520</y>
        <width>86</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Under Attack
(enabled)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="icon">
      <iconset>
       <normaloff>img/dns.png</normaloff>img/dns.png</iconset>
     </attribute>
     <attribute name="title">
      <string>DNS Query</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditDomainQuery">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>690</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="placeholderText">
       <string>Domain</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRecordType_2">
      <property name="geometry">
       <rect>
        <x>712</x>
        <y>10</y>
        <width>69</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <item>
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AAAA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CNAME</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TXT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SRV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CAA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NS</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButtonQuery">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Query</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowserResult">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>881</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset>
       <normaloff>img/ip-replace.png</normaloff>img/ip-replace.png</iconset>
     </attribute>
     <attribute name="title">
      <string>DNS IP Replacer</string>
     </attribute>
     <widget class="QLineEdit" name="txtNewIP">
      <property name="geometry">
       <rect>
        <x>539</x>
        <y>5</y>
        <width>250</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>New IP Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtOldIP">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>5</y>
        <width>250</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Old IP Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>9</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Old IP Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>439</x>
        <y>9</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>New IP Address</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>38</y>
        <width>884</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReplace">
      <property name="geometry">
       <rect>
        <x>815</x>
        <y>5</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Replace</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>49</y>
        <width>884</width>
        <height>513</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>428</x>
        <y>5</y>
        <width>3</width>
        <height>24</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="icon">
      <iconset>
       <normaloff>img/whois.png</normaloff>img/whois.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Whois Query</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditDomainWhois">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>770</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="placeholderText">
       <string>Domain</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonWhois">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>100</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Whois</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>877</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowserWhoisResult">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>881</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="icon">
      <iconset>
       <normaloff>img/settings.png</normaloff>img/settings.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>541</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>CloudFlare API Settings</string>
      </property>
      <widget class="QLineEdit" name="txtKey">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>80</y>
         <width>500</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>CloudFlare API Key</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>241</x>
         <y>160</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtEmail">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>120</y>
         <width>500</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>CloudFlare Email</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>541</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>DNS Query Resolver</string>
      </property>
      <widget class="QPushButton" name="pushButtonResolverAdd">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>20</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidgetResolver">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>210</width>
         <height>181</height>
        </rect>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>125</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Resolver IP</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>id</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButtonResolverRemove">
       <property name="geometry">
        <rect>
         <x>245</x>
         <y>170</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditResolver">
       <property name="geometry">
        <rect>
         <x>245</x>
         <y>20</y>
         <width>192</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
